docker-compose_prod.yml

```diff
@@ -0,0 +1,3 @@
+services:
+  app:
+    command: ./harvest-admin-start.sh
```

Comment:
This isn't actually used for production, right? Is this used to simulate our production startup on cloud.gov?

Reply:
Correct